### DISCUSSION TITLE
What is the correct syntax to apply a format to a date using handlebar expressions please?

### DISCUSSION THREAD

**Discussion by steveh2023** [microsoft-search/pnp-modern-search#3468] [Q&A] [answered]
Hi,

Here is the handlebar sharepoint field name:

{{0DE-Merlin-PublicationDate}} 

and this is the date format:

{{{dateFormat "ll" timeStamp}}} 

How do i put them together so that the format is applied to the date please?

I tried this:
 
{{0DE-Merlin-PublicationDate}}{{{dateFormat "ll" timeStamp}}} 

and i get: 

"Error: Missing helper: "dateFormat""

**Comment by wobba** [comment, marked as answer]
See `getDate` in the docs https://microsoft-search.github.io/pnp-modern-search/extensibility/templating/#handlebars-helpers And it uses date formatting same as the moment library.

**Reply by steveh2023** [reply]
Thanks!
